API/NoDocs/Aspire.Microsoft.Azure.Cosmos/AzureCosmosDBSettings.cs

```diff
@@ -0,0 +1,16 @@
+// Assembly 'Aspire.Microsoft.Azure.Cosmos'
+
+using System;
+using System.Runtime.CompilerServices;
+using Azure.Core;
+
+namespace Aspire.Microsoft.Azure.Cosmos;
+
+public sealed class AzureCosmosDBSettings
```

Comment:
We should _consider_ including `Microsoft` in this name.  We're still using the Track 1 Cosmos client which starts with `Microsoft.*` and while we could still use `AzureCosmosSettings` for a future Track 2 client it might be a little confusing to have both of these.  Ditto on the extension method to add a client.

Reply:
Add Microsoft